### ISSUE TITLE
Got a parsing errors when [yum install] with the 2.3.1809.x86_64 images

### ISSUE THREAD

**Issue by zhenweiding** [euleros/euleros-docker-images#13]
I got this parsing error when yum install with the image:
euleros-docker-images/2.3.1809/x86_64/EulerOS-2.3-x86_64.tar.xz

![Screen Shot 2019-03-27 at 1 28 39 PM](https://user-images.githubusercontent.com/4710755/55054608-71bee680-509b-11e9-8feb-0a8ced3da86f.png)

This error can be fixed by change the [/etc/yum.repos.d/euleros.repo]'s content to this:
![Screen Shot 2019-03-27 at 2 11 45 PM](https://user-images.githubusercontent.com/4710755/55054724-c2364400-509b-11e9-86cb-3f316a3ef452.png)

Is this "baseurl" correct? or please give me a right url!

**Comment by Pennyzct** [comment]
Hi~ @aarch64 @odidev 
this error also occurred in aarch64. And it existed both in `2.3.1803` and `2.3.1809`.
```
bash-4.2# yum -y update


File contains parsing errors: file:///etc/yum.repos.d/euleros.repo
        [line  2]:  name=EulerOS-2.0SP3 base

        [line  3]:  baseurl=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.3/os//

        [line  4]:  enabled=1

        [line  5]:  gpgcheck=1

        [line  6]:  gpgkey=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.3/os/RPM-GPG-KEY-EulerOS

```
probably, we can do some thing like  `echo baseurl=file://${ISO_PATH}/${machine} >> ${euleros_repo}` in [generate.sh](https://github.com/euleros/euleros-docker-images/blob/master/scripts/generate.sh#L44).

**Comment by liusdu** [comment]
Hi  @zhenweiding  Sorry for this mistake,  There is something wrong with my automatic scripts..
Because docker hub  belongs to American company. So maybe I can not push the correct images right now.. So please change this file with the following

* X86_64

```
[base]
name=EulerOS-2.0SP3 base
baseurl=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.3/os/x86_64/
enabled=1
gpgcheck=1
gpgkey=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.3/os/RPM-GPG-KEY-EulerOS
```


* ARM64

```
[base]
name=EulerOS-2.0SP3 base
baseurl=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.3/os/aarch64/
enabled=1
gpgcheck=1
gpgkey=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.3/os/RPM-GPG-KEY-EulerOS

```

**Comment by zhenweiding** [comment]
@liusdu thanks, i see!

**Comment by pergolafabio** [comment]
hi, i had the same issue indeed, but seems the repos are now forbidden? i tried 2.2 and 2.3

btw, when i try 2.3 image, how can i change the file , there is no editor included? vi / vim / nano not included, while its in 2.2

![image](https://user-images.githubusercontent.com/43251136/202677941-db163558-e5f6-418c-9752-702c97c5e647.png)
